docs/src/getting-started/tutorial-setup.md

```diff
@@ -15,7 +15,9 @@ If you're looking for a very fast (but less comprehensive) "first contact" exper
 
 The tutorial assumes you have some basic knowledge of what Zeebe is and what it's used for. If you're completely new to Zeebe, you might find it helpful to read through the ["What is Zeebe?"](https://docs.zeebe.io/introduction/what-is-zeebe.html) docs article first.
 
-Below are the components you'll use in the tutorial. Please download the full distributions for these components instead of running them with Docker.
+Below are the components you'll use in the tutorial. The easiest way to run them is to use the `operate` docker-compose profile in the [zeebe-docker-compose](https://github.com/zeebe-io/zeebe-docker-compose) repository.
```

Comment:
Even if using docker-compose, is it still necessary to download the Modeler distribution?

Reply:
I've updated the tutorial.

Reply:
WDYT?